src/main/User/User.java

```diff
@@ -347,6 +355,7 @@ private static UserValidationMessage isValid(
       log.error("Invalid password: " + password);
       return UserValidationMessage.INVALID_PASSWORD;
     }
+    // insert validation for defaultIds
```

Comment:
here would be the way you would validate the string and turn them into enums

Reply:
Update, I dont think validation is needed for User.java because we are initializing the user with an empty hashmap. No validation needs to be done on user initialization